### ISSUE TITLE
Reporting errors for TS-interface

### ISSUE THREAD

**Issue by J-Kallunki** [sweepline/eslint-plugin-unused-imports#16]
## Config:
```js
    'unused-imports/no-unused-imports-ts': 'error',
        'unused-imports/no-unused-vars-ts': [
          'error',
          {
            vars: 'all',
            varsIgnorePattern: '^_',
            args: 'after-used',
            argsIgnorePattern: '^_',
          },
        ],
```

## Code
```ts
export interface SampleProps {
  entryFunc?: (id?: number) => void;
}
```

## ERROR:
```bash
error  'id' is defined but never used. Allowed unused args must match /^_/u  unused-imports/no-unused-vars-ts
```

**Comment by sweepline** [comment]
I can't reproduce this. Did you also do `"@typescript-eslint/no-unused-vars": "off"`?

**Comment by J-Kallunki** [comment]
Yes also these:
```json
        '@typescript-eslint/no-unused-vars': 'off',
        'unused-imports/no-unused-imports': 'off',
        'unused-imports/no-unused-vars': 'off',
```

And it is defined with overrides
```
